src/componets/DisplayDate.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { useEffect, useState} from "react"; 
+
+const DisplayDate = ()=> {
+
+    const [date, setDate] = useState()
+
+    useEffect (() =>{
+         getDate()
+      },[])
+
+    function getDate () {
+        const today = new Date();
+        const month = today.getMonth() + 1;
+        const year = today.getFullYear();
+        const date = today.getDate();
+        setDate(`${month}/${date}/${year}`)
```

Comment:
QA Team - Date is in the American Format, According to requirements should be in English format.